app/src/main/java/com/ggetters/app/data/repository/lineup/CombinedLineupRepository.kt

```
@@ -48,8 +71,242 @@ class CombinedLineupRepository @Inject constructor(
         }
     }
 
-    override fun getByEventId(eventId: String): Flow<List<Lineup>> =
-        offline.getByEventId(eventId)
+    override fun getByEventId(eventId: String): Flow<List<Lineup>> = channelFlow {
```

detekt (1.23.8)

[warning] The function getByEventId appears to be too complex based on Cyclomatic Complexity (complexity: 34). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)